webpack.config.js

```diff
@@ -0,0 +1,31 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  entry: './src/index.js',
+  devServer: {
+    static: './dist',
+  },
```

Comment:
Webpack compiles with errors as shown below 👇 
![image](https://user-images.githubusercontent.com/46839548/193091747-86135645-19d7-4ad3-8e56-87149c8b0dea.png)

- [ ] Please set the mode to 'development' to prevent webpack from compiling with errors like this 👇 
``` javascript
module.exports = {
  entry: './src/index.js',
  mode: 'development',
  devServer: {
    static: './dist',
  },
.
.
.
```

Reply:
Done 👍 
![image](https://user-images.githubusercontent.com/11477535/193106635-b670f512-40ec-4a36-9954-18adf29d5498.png)